kolibri/core/assets/src/views/k-dropdown-menu/index.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+
+  <div class="dib">
+    <!-- Added because the trigger could not be a vue element smh-->
+    <div
+      ref="buttonContainer"
+      class="button-container dib"
+    >
+      <k-button
+        ref="button"
+        :text="text"
+        :appearance="appearance"
+        :primary="primary"
+        :disabled="disabled"
+        :hasDropdown="true"
+      />
+    </div>
+
+    <ui-popover
+      ref="popover"
+      trigger="buttonContainer"
+      :containFocus="false"
+      :position="position"
+      @close="handleClose"
+    >
+      <ui-menu
+        :options="options"
+        @select="emitSelection"
+      />
+    </ui-popover>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import kButton from 'kolibri.coreVue.components.kButton';
+  import UiPopover from 'keen-ui/src/UiPopover';
+  import uiMenu from 'keen-ui/src/UiMenu';
+
+  export default {
+    name: 'kDropdownMenu',
+    components: {
+      kButton,
+      UiPopover,
+      uiMenu,
+    },
+    props: {
+      text: {
+        type: String,
+        required: true,
+      },
+      appearance: {
+        type: String,
+        required: false,
+        default: 'raised-button',
```

Comment:
you can add a validator here, re-using this one: https://github.com/learningequality/kolibri/pull/3257/files#diff-6f198d4d8621b85cdf03813e2db84ee6R24